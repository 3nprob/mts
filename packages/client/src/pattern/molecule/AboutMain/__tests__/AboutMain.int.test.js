import { regular } from './AboutMain.int.story'

describe('<AboutMain />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
