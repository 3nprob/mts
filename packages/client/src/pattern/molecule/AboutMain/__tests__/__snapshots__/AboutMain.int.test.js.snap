// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AboutMain /> Snaphot must match defaultRender 1`] = `
<main
  className=""
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
      "marginTop": "55px",
    }
  }
>
  <section
    className="container1"
  >
    <header
      className="container1Header"
    >
      <h1>
        About Me
      </h1>
      <meta
        content="About Me"
        itemProp="name"
      />
      <span
        className="container1HeaderSubHeader"
      >
        <div
          className="container1HeaderSubHeaderPrimary"
        >
          <meta
            content="Software Engineer for React.js, Node.js, GraphQL and JavaScript. English speaking. Consulting/Freelancing for Web Development project: Code Audits/Reviews, Workshops, Training, Implementation ..."
            itemProp="description"
          />
          <time
            dateTime="01-01-2020"
            itemProp="datePublished"
          >
            January 1, 2020
          </time>
          <meta
            content="01-01-2020"
            itemProp="dateModified"
          />
          <meta
            content="american react consultant,american react freelancer,american javascript freelancer,american javascript consultant"
            itemProp="keywords"
          />
          <meta
            content="2384"
            itemProp="wordCount"
          />
          <span>
             by 
            <span>
              MTS
            </span>
            <meta
              content="MTS"
              itemProp="author"
            />
          </span>
          <div
            itemProp="publisher"
            itemScope=""
            itemType="https://schema.org/Organization"
          >
            <meta
              itemProp="name"
            />
            <div
              itemProp="logo"
              itemScope=""
              itemType="https://schema.org/ImageObject"
            >
              <meta
                content="https://mts.github.io/mts/static/logo.jpg"
                itemProp="url"
              />
            </div>
          </div>
          <span>
             - 
            <a
              href="https://github.com/mts/mts"
              rel="noopener noreferrer"
              target="_blank"
            >
              Edit this Post
            </a>
          </span>
        </div>
      </span>
      <ul
        className="container1HeaderFollowButtons"
      >
        <li
          className="container1HeaderFollowButtonsItem"
        >
          <a
            href="https://twitter.com/hashtag/willsmith"
            rel="noopener noreferrer"
            target="_blank"
          >
            <img
              alt="Follow on Twitter"
              src="https://img.shields.io/twitter/follow/willsmith.svg?style=social&label=Follow on Twitter"
            />
          </a>
        </li>
        <li
          className="container1HeaderFollowButtonsItem"
        >
          <a
            className="container1HeaderFollowButtonsItemFacebookFollow"
            href="https://www.facebook.com/Will-Smith-92304305160/"
            rel="noopener noreferrer"
            target="_blank"
          >
            <svg
              height="12px"
              viewBox="0 0 96.124 96.123"
              width="12px"
            >
              <path
                d="M72.089.02L59.624 0C45.62 0 36.57 9.285 36.57 23.656v10.907H24.037a1.96 1.96 0 0 0-1.96 1.961v15.803a1.96 1.96 0 0 0 1.96 1.96H36.57v39.876a1.96 1.96 0 0 0 1.96 1.96h16.352a1.96 1.96 0 0 0 1.96-1.96V54.287h14.654a1.96 1.96 0 0 0 1.96-1.96l.006-15.803a1.963 1.963 0 0 0-1.961-1.961H56.842v-9.246c0-4.444 1.059-6.7 6.848-6.7l8.397-.003a1.96 1.96 0 0 0 1.959-1.96V1.98A1.96 1.96 0 0 0 72.089.02z"
              />
            </svg>
             Follow on Facebook
          </a>
        </li>
      </ul>
      <div
        className="container1HeaderContainer1"
      >
        <div
          style={
            Object {
              "paddingBottom": "75%",
              "width": "100%",
            }
          }
        />
        <picture>
          <img
            alt="banner"
            src={Object {}}
          />
        </picture>
      </div>
      <meta
        content={Object {}}
        itemProp="image"
      />
    </header>
    <meta
      content="https://mts.github.io/about"
      itemProp="mainEntityOfPage"
    />
    <arcticle
      itemProp="articleBody"
    >
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie. Morbi ac dapibus urna. Donec magna felis, porta eget felis vel, sodales fringilla quam. Maecenas mauris ligula, tempus et pretium quis, aliquam nec odio. Praesent mattis a leo lacinia facilisis. Integer rutrum ac purus nec blandit. Nullam et felis ut eros sodales laoreet non vel ex. Suspendisse quis viverra purus. Pellentesque lobortis semper laoreet. Proin venenatis lacus nec neque aliquam cursus. Nullam porttitor a nulla a venenatis. Duis eu sapien velit. Phasellus sagittis vitae ligula eget posuere. Nullam volutpat cursus nunc, at molestie tellus pellentesque non. Vestibulum dignissim ante vel diam tincidunt iaculis. Vestibulum vehicula lobortis ultrices.
      </p>
      <h2
        className="container1Title"
      >
        <a
          aria-label="values and mission permalink"
          href="#values-and-mission"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Values and Mission
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie. Morbi ac dapibus urna. Donec magna felis, porta eget felis vel, sodales fringilla quam. Maecenas mauris ligula, tempus et pretium quis, aliquam nec odio. Praesent mattis a leo lacinia facilisis. Integer rutrum ac purus nec blandit. Nullam et felis ut eros sodales laoreet non vel ex. Suspendisse quis viverra purus. Pellentesque lobortis semper laoreet. Proin venenatis lacus nec neque aliquam cursus. Nullam porttitor a nulla a venenatis. Duis eu sapien velit. Phasellus sagittis vitae ligula eget posuere. Nullam volutpat cursus nunc, at molestie tellus pellentesque non. Vestibulum dignissim ante vel diam tincidunt iaculis. Vestibulum vehicula lobortis ultrices.
      </p>
      <h2
        className="container1Title"
      >
        <a
          aria-label="passions beyond software development permalink"
          href="#passions-beyond-software-development"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Passions beyond Software Development
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h2
        className="container1Title"
      >
        <a
          aria-label="you like my content here are a couple of ways to support it permalink"
          href="#you-like-my-content-here-are-a-couple-of-ways-to-support-it"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        You like my content? Here are a couple of ways to support it.
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt. Curabitur commodo, sapien id mattis sagittis, lectus dolor ullamcorper purus, et lacinia elit urna tincidunt massa. Nam pretium nisi at vehicula placerat. Ut molestie eros in sapien mollis, tincidunt vehicula arcu pellentesque. Proin vel elementum ante. Curabitur ut erat massa. Integer pellentesque mauris sed libero scelerisque molestie.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="be my patron permalink"
          href="#be-my-patron"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Be my Patron
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="follow me permalink"
          href="#follow-me"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Follow Me
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="leave a review permalink"
          href="#leave-a-review"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Leave a Review
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="share permalink"
          href="#share"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Share
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="learn react permalink"
          href="#learn-react"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Learn React
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="contribute permalink"
          href="#contribute"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Contribute
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="donate permalink"
          href="#donate"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Donate
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <h2
        className="container1Title"
      >
        <a
          aria-label="what i use permalink"
          href="#what-i-use"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        What I Use
      </h2>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante. Cras tortor nisi, vulputate in pharetra et, cursus non felis. Pellentesque libero nulla, auctor ac viverra eu, gravida eu odio. Vestibulum dolor ex, mollis eget euismod et, auctor ut est. Etiam posuere hendrerit tincidunt.
      </p>
      <h3>
        <a
          aria-label="heroku and github pages permalink"
          href="#heroku-and-github-pages"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Heroku and GitHub Pages
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <h3>
        <a
          aria-label="email permalink"
          href="#email"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Email
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
      </p>
      <h3>
        <a
          aria-label="editor and terminal permalink"
          href="#editor-and-terminal"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Editor and Terminal
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <h3>
        <a
          aria-label="nomad gear permalink"
          href="#nomad-gear"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Nomad Gear
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
      </p>
      <ul
        className="container1List"
      >
        <li
          className="container1ListItem"
        >
          <a
            href="https://amzn.to/2RwCPwg"
            rel="noopener noreferrer"
            target="_blank"
          >
            Trackpad
          </a>
           
          and
           
          <a
            href="https://amzn.to/2SPqZ1K"
            rel="noopener noreferrer"
            target="_blank"
          >
            Keyboard
          </a>
           
        </li>
        <li
          className="container1ListItem"
        >
          <a
            href="https://amzn.to/2RChPEq"
            rel="noopener noreferrer"
            target="_blank"
          >
            Display
          </a>
           
        </li>
      </ul>
      <h3>
        <a
          aria-label="apps gear permalink"
          href="#apps"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Apps
      </h3>
      <ul
        className="container1List"
      >
        <li
          className="container1ListItem"
        >
          <a
            href="https://apps.apple.com/us/app/twitter/id333903271"
            rel="noopener noreferrer"
            target="_blank"
          >
            Twitter
          </a>
        </li>
      </ul>
      <h3>
        <a
          aria-label="freelancers permalink"
          href="freelancers"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Freelancers
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <ul
        className="container1List"
      >
        <li
          className="container1ListItem"
        >
          <p>
            <strong>
              Online Marketing and Social Media:
            </strong>
             
            <p>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
            </p>
          </p>
        </li>
        <li
          className="container1ListItem"
        >
          <p>
            <strong>
              Proofreader/Editor:
            </strong>
            <p>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
            </p>
          </p>
        </li>
        <li
          className="container1ListItem"
        >
          <p>
            <strong>
              Logo Design:
            </strong>
             I worked with 
            <p>
              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
            </p>
          </p>
        </li>
      </ul>
      <h3>
        <a
          aria-label="revue permalink"
          href="revue"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Revue
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <ul
        className="container1List"
      >
        <li
          className="container1ListItem"
        >
          permalinks to newsletters
        </li>
        <li
          className="container1ListItem"
        >
          powerful integrations for fetching content automatically
        </li>
        <li
          className="container1ListItem"
        >
          simple and intuitive UI/UX
        </li>
        <li
          className="container1ListItem"
        >
          great pricing model for a larger audience
        </li>
        <li
          className="container1ListItem"
        >
          statistics and analytics
        </li>
      </ul>
      <h3>
        <a
          aria-label="leanpub permalink"
          href="leanpub"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Leanpub
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <h3>
        <a
          aria-label="educative permalink"
          href="educative"
          rel="noopener noreferrer"
          target="_blank"
        >
          <svg
            height="0"
            width="0"
          />
        </a>
        Educative
      </h3>
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien. Sed quis purus quis metus sagittis sollicitudin accumsan in ante.
      </p>
      <hr
        className="container1Divider"
      />
      <p>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent nec turpis eleifend, pretium leo ut, euismod sapien.
      </p>
    </arcticle>
    <div
      className="container1DividerButton"
    >
      <button
        type="button"
      >
        Show Comments
      </button>
    </div>
  </section>
</main>
`;
